src/test/java/org/opentripplanner/street/model/edge/VehicleRentalEdgeTest.java

```diff
@@ -0,0 +1,195 @@
+package org.opentripplanner.street.model.edge;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.framework.i18n.NonLocalizedString;
+import org.opentripplanner.routing.algorithm.GraphRoutingTest;
+import org.opentripplanner.routing.api.request.StreetMode;
+import org.opentripplanner.routing.api.request.request.VehicleRentalRequest;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.service.vehiclerental.model.RentalVehicleType;
+import org.opentripplanner.service.vehiclerental.model.VehicleRentalStation;
+import org.opentripplanner.service.vehiclerental.street.VehicleRentalEdge;
+import org.opentripplanner.service.vehiclerental.street.VehicleRentalPlaceVertex;
+import org.opentripplanner.street.model.RentalFormFactor;
+import org.opentripplanner.street.search.request.StreetSearchRequest;
+import org.opentripplanner.street.search.state.State;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+
+class VehicleRentalEdgeTest extends GraphRoutingTest {
```

Comment:
```suggestion
class VehicleRentalEdgeTest {
```

Do you need the GraphRoutingTest?

Reply:
Well spotted. Seems like it's not needed. I used another test class as the basis for mine but it was unnecessary in the other class as well so I removed it from both.